lib/github/templates/security-post-release.md

```diff
@@ -0,0 +1,18 @@
+---
+date: %ANNOUNCEMENT_DATE%
+category: vulnerability
+title:  %RELEASE_DATE% Security Releases
+slug: %SLUG%
+layout: blog-post.hbs
+author: %AUTHOR%
+---
+
+## Security releases available
+
+Updates are now available for the %AFFECTED_VERSIONS% Node.js release lines for the
+following issues.
+%DEPENDENCY_UPDATES%
+%OPENSSL_UPDATES%
```

Comment:
Why OpenSSL != Dependecy updates

Reply:
In the past releases we had different sections, since on the openssl one we write advisories

Reply:
I think we can fit OpenSSL into dependency updates to avoid adding more complexity to the code.